week3/57-skhiema-gorniera/task.py

```diff
@@ -0,0 +1,15 @@
+def main():
+    n, x = int(input()), float(input())
+    prev_part = 0
+    for i in range(n):
+        a = float(input())
+        prev_part = (prev_part + a) * x
+    prev_part += float(input())
```

Comment:
why not part of main cycle?

Reply:
because for the last term multiplication by x is not required, unlike the rest